training/models/event_event.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+from openerp import api, fields, models
+from .common import M
+from .. import exceptions
+
+
+class Event(models.Model):
+    """Expand events with training actions.
+
+    Events with a training type and a training action are considered training
+    groups.
+    """
+    _inherit = "event.event"
+
+    material_ids = fields.Many2many(
+        M % "material",
```

Comment:
I find this unnecessary, and can't see a clear benefit for it.
What is the problem with just using `training.material`?